packages/teleport/src/Databases/DatabaseList/DatabaseList.tsx

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { sortBy } from 'lodash';
+import { Flex, ButtonBorder } from 'design';
+import {
+  Column,
+  SortHeaderCell,
+  Cell,
+  renderLabelCell,
+  TextCell,
+  SortTypes,
+} from 'design/DataTable';
+import Table from 'design/DataTable/Paged';
+import isMatch from 'design/utils/match';
+import { AuthType } from 'teleport/services/user';
+import InputSearch from 'teleport/components/InputSearch';
+import { Database, DbProtocol } from 'teleport/services/databases';
+import ConnectDialog from 'teleport/Databases/ConnectDialog';
+
+function DatabaseList(props: Props) {
+  const { databases = [], pageSize = 20, user, clusterId, authType } = props;
+
+  const [sortDir, setSortDir] = useState<Record<string, string>>({
+    name: SortTypes.DESC,
+  });
+
+  const [searchValue, setSearchValue] = useState<string>('');
```

Comment:
need to move the search value 1 level up, as it needs to stay when a user navigates between clusters.